src/jit/codegenxarch.cpp

```diff
@@ -1813,162 +1813,20 @@ CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
                     assert(structDesc.eightByteCount == CLR_SYSTEMV_MAX_EIGHTBYTES_COUNT_TO_PASS_IN_REGISTERS);
 
                     regNumber retReg0 = REG_NA;
-                    emitAttr size0 = EA_UNKNOWN;
-                    unsigned offset0 = structDesc.eightByteOffsets[0];
+                    unsigned __int8 offset0 = 0;
```

Comment:
Another issue that i have noticed in GT_RETURN:  While generating Profiler Leave callback codegen artificially extends the life of RAX if the return type is a GC ref/byref.  When we enable profile callbacks for Unix amd64, in case of struct returned in two registers, we would have to extend the life of RAX and RDX.

Not sure whether profile Enter/Leave/Tailcall callbacks are enabled/supported on unix.  In either case we should open an issue to track this.